util/topgen/intermodule.py

```diff
@@ -521,47 +521,42 @@ def check_intermodule_field(obj: OrderedDict, prefix: str = "") -> int:
     return error
 
 
-def find_otherside_modules(topcfg: OrderedDict, m,
-                           s) -> List[Tuple[str, str, str]]:
+def find_otherside_module(topcfg: OrderedDict, m, s) -> Tuple[str, str, str]:
```

Comment:
@eunchan 
I made some change in this function
1. don't need to return an list, as only module is expected
2. handle mems here as they're not comportable IP now
3. comparing with `top_signame` seems not needed since we need port name instead of interconnect signal name

Reply:
I recommend it to return the list as I expect this function to be used in somewhere else. For multi width inter-module signal, it should have a list.

Reply:
OK, change it back to list